app/controllers/concerns/hyrax/works_controller_behavior.rb

```diff
@@ -189,36 +175,38 @@ def actor
 
     ##
     # @return [#errors]
-    def create_work
-      case curation_concern
-      when ActiveFedora::Base
-        actor.create(actor_environment)
-      else
-        form = build_form
-
-        @curation_concern =
-          form.validate(params[hash_key_for_curation_concern]) &&
```

Comment:
Setting the work to the results of `form.validate` upon failure is the cause of issue #5465.